{
	"class": "patch",
	"id": "oval:com.ubuntu.xenial:def:46641000000",
	"version": "1",
	"metadata": {
		"title": "USN-4664-1 -- Aptdaemon vulnerabilities",
		"description": "Kevin Backhouse discovered that Aptdaemon incorrectly handled certain properties. A local attacker could use this issue to test for the presence of local files. (CVE-2020-16128)  Kevin Backhouse discovered that Aptdaemon incorrectly handled permission checks. A local attacker could possibly use this issue to cause a denial of service. (CVE-2020-27349)  Update Instructions:  Run `sudo pro fix USN-4664-1` to fix the vulnerability. The problem can be corrected by updating your system to the following package versions:  python3-aptdaemon.gtk3widgets - 1.1.1+bzr982-0ubuntu14.5 aptdaemon-data - 1.1.1+bzr982-0ubuntu14.5 python3-aptdaemon.test - 1.1.1+bzr982-0ubuntu14.5 python-aptdaemon - 1.1.1+bzr982-0ubuntu14.5 aptdaemon - 1.1.1+bzr982-0ubuntu14.5 python3-aptdaemon.pkcompat - 1.1.1+bzr982-0ubuntu14.5 python-aptdaemon.gtk3widgets - 1.1.1+bzr982-0ubuntu14.5 python3-aptdaemon - 1.1.1+bzr982-0ubuntu14.5 No subscription required",
		"affected": {
			"family": "unix",
			"platform": "Ubuntu 16.04 LTS"
		},
		"reference": [
			{
				"source": "USN",
				"ref_id": "USN-4664-1",
				"ref_url": "https://ubuntu.com/security/notices/USN-4664-1"
			},
			{
				"source": "CVE",
				"ref_id": "CVE-2020-16128",
				"ref_url": "https://ubuntu.com/security/CVE-2020-16128"
			},
			{
				"source": "CVE",
				"ref_id": "CVE-2020-27349",
				"ref_url": "https://ubuntu.com/security/CVE-2020-27349"
			}
		],
		"advisory": {
			"from": "security@ubuntu.com",
			"severity": "Medium",
			"issued": {
				"date": "2020-12-08"
			},
			"cve": [
				{
					"text": "CVE-2020-16128",
					"href": "https://ubuntu.com/security/CVE-2020-16128",
					"priority": "medium",
					"public": "20201208",
					"cvss_score": "3.8",
					"cvss_vector": "CVSS:3.1/AV:L/AC:L/PR:L/UI:N/S:C/C:L/I:N/A:N",
					"cvss_severity": "low",
					"usns": "4664-1"
				},
				{
					"text": "CVE-2020-27349",
					"href": "https://ubuntu.com/security/CVE-2020-27349",
					"priority": "medium",
					"public": "20201208",
					"cvss_score": "5.5",
					"cvss_vector": "CVSS:3.1/AV:L/AC:L/PR:L/UI:N/S:U/C:N/I:N/A:H",
					"cvss_severity": "medium",
					"usns": "4664-1"
				}
			]
		}
	},
	"criteria": {
		"criterias": [
			{
				"operator": "OR",
				"criterions": [
					{
						"test_ref": "oval:com.ubuntu.xenial:tst:466410000000",
						"comment": "Long Term Support"
					}
				]
			}
		]
	}
}
